plugin/vcs/vcs.go

```diff
@@ -117,6 +133,12 @@ type Provider interface {
 	// repositoryID: the repository ID from the external VCS system (note this is NOT the ID of Bytebase's own repository resource)
 	FetchRepositoryActiveMemberList(ctx context.Context, oauthCtx common.OauthContext, instanceURL string, repositoryID string) ([]*RepositoryMember, error)
 
+	// Fetch all repository in which the authenticated user has a maintainer role
```

Comment:
Sorry. The last comment here is not accurate. The interface should be general to just say returning the repository list. While the GitLab implementation should mention the maintainer role part.